frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -416,11 +416,13 @@ function BookInfo:setCustomMetadata(file, book_props, prop_key, prop_value)
     -- in memory
     prop_value = prop_value or custom_doc_settings:readSetting("doc_props")[prop_key] -- set custom or restore original
     book_props[prop_key] = prop_value
+    if prop_key == "title" then -- generate if original is empty
```

Comment:
`-- generate if original is empty`

Shouldn't it be: if _updated_ title is empty ?

Reply:
We cannot have customized title empty.

Reply:
So, `-- generate when customization reset and original is empty` ?